include/UIKit/UIApplicationDelegate.h

```diff
@@ -36,17 +36,18 @@
 /* TODO 7328699::
  * Projection headers cannot currenly be included in modules, so we must forward declare classes for now
  * App developers should import the following files to use methods requiring projected classes
- * #import <UWP/WindowsFoundation.h>
- * #import <UWP/WindowsMediaSpeechRecognition.h>
+ * #import <UWP/WindowsApplicationModelActivation.h>
```

Comment:
Currently we can only use forward-declared classes, but it's non ideal as developers will need to include any projection headers in their code rather than just <UIKit.h>

---
In reply to: [70539688](https://github.com/Microsoft/WinObjC/pull/637#discussion_r70539688) [](ancestors = 70539688)